ext/web/03_abort_signal.js

```diff
@@ -21,51 +20,57 @@
   const add = Symbol("add");
   const signalAbort = Symbol("signalAbort");
   const remove = Symbol("remove");
+  const aborted = Symbol("aborted");
+  const abortAlgos = Symbol("abortAlgos");
 
   const illegalConstructorKey = Symbol("illegalConstructorKey");
 
   class AbortSignal extends EventTarget {
-    #aborted = false;
-    #abortAlgorithms = new Set();
```

Comment:
In the future, it would be good to add comments about performance optimizations so someone doesn't accidentally do something to cause a regression. For example, stating here not to use private properties on this class.

Reply:
This change will probably have a side effect of leaking these fields during object inspection - AFAIK private fields do not show up during inspection. I'm not sure though if it is against the spec or anything.

Reply:
> For example, stating here not to use private properties on this class.

Well for Web APIs we don't use private properties anywhere, but only symbols. so this was just old code that wasn't updated yet to how we do things.